src/Resources/MeshId.h

```diff
@@ -9,6 +9,8 @@
 
 #pragma once
 
+#include <entt/entt.hpp>
```

Comment:
entt/core/hashed_string.hpp

Reply:
> Is it a good idea to have the Abstract base classes in the same headers as the default provided definitions?

No it isn't. We don't have multiple implementations and we have forward declarations now so there's no performance to be gained.

Might take a bit of work to do but probably best to do it now before we have 100 systems.

Reply:
> entt/core/hashed_string.hpp

Thanks

Reply:
> > Is it a good idea to have the Abstract base classes in the same headers as the default provided definitions?
> 
> No it isn't. We don't have multiple implementations and we have forward declarations now so there's no performance to be gained.
> 
> Might take a bit of work to do but probably best to do it now before we have 100 systems.

@bwrsandman #492